### PR TITLE
Attempt to fix /economystats + add an alias for a command

### PR DESCRIPTION
**Pull request by ghost**
Here goes nothing...

Keep in mind that I'm currently on an iPad so I can't test anything atm >_>

### CODE REVIEW

File: spacialgaze-plugins/economy.js [hunk 1]
```diff
@@ -201,6 +201,7 @@ exports.commands = {
 		});
 	},
 
+	transfer: 'transferstardust',
```
Comment (HoeenCoder) [inline spacialgaze-plugins/economy.js]: Id perfer to save this alias for SGgame

File: spacialgaze-plugins/economy.js [hunk 2]
```diff
@@ -321,11 +322,12 @@ exports.commands = {
 		this.sendReply('Your symbol has been removed.');
 	},
 
+	economy: 'economystats',
 	currency: 'economystats',
 	stardust: 'economystats',
 	economystats: function (target, room, user) {
 		if (!this.runBroadcast()) return;
-		const users = Object.keys(Db.currency.object());
+		const users = Object.keys(Db.currency.object);
```
Comment (HoeenCoder) [inline spacialgaze-plugins/economy.js]: Dosent exists, try modifying it to work with `Db.currency.keys()` which is an array of every entry in the db (all userids in this case)

File: spacialgaze-plugins/economy.js (hunk 1 above)
Comment (ghost) [reply spacialgaze-plugins/economy.js]: SO THATS WHY IT WASNT USED!!!!!1!1!11!1!11!one!!1!1!1!1!!!!1!!11111!eleven!!!

File: spacialgaze-plugins/economy.js (hunk 2 above)
Comment (ghost) [reply spacialgaze-plugins/economy.js]: Ok lets try this

**Comment by ghost** [comment]
Failed Testing >_>

Im not looking to move forward until i have some help with this

**Comment by Mystifi** [comment]
I'll fix it tomorrow, I know how to do so.

**Comment by ghost** [comment]
Perfect! @Mystifi can you also add the economy alias like I did?

**Comment by Mystifi** [comment]
Yeah, definitely.

**Pull request closed without merging**